package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "react_mp",
+  "version": "1.0.0",
+  "description": "ReactJS Mentoring Programm",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack && nodemon server.js",
+    "dev": "webpack-dev-server",
+    "build": "webpack",
+    "server": "nodemon server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/evgeny-sav/react_mp.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/evgeny-sav/react_mp/issues"
+  },
+  "homepage": "https://github.com/evgeny-sav/react_mp#readme",
+  "dependencies": {
+    "axios": "^0.16.2",
+    "bootstrap": "^4.0.0-beta",
+    "cors": "^2.8.4",
```

Comment:
We need it. Otherwise we get an error
```
No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'http://localhost:8080' is therefore not allowed access.
```